src/SFA.DAS.EmploymentCheck.Functions/Application/Clients/EmployerAccount/EmployerAccountClient.cs

```diff
@@ -24,6 +24,10 @@ public EmployerAccountClient(IEmployerAccountService employerAccountService)
             foreach (var employmentCheck in employmentChecksBatch)
             {
                 var employerPayeSchemes = await _employerAccountService.GetEmployerPayeSchemes(employmentCheck);
+                for (int i = 0; i < employerPayeSchemes?.PayeSchemes?.Count; i++)
+                {
+                    employerPayeSchemes.PayeSchemes[i] = employerPayeSchemes.PayeSchemes[i].Trim().ToUpper();
```

Comment:
is this change meant to be part of this ticket?

Reply:
It was added to fix a failing test but as per discussion have now merged from master and the test succeeds so have removed the code.